packages/features/calendar-cache/lib/getShouldServeCache.ts

```diff
@@ -0,0 +1,8 @@
+import { FeaturesRepository } from "@calcom/features/flags/features.repository";
+
+export async function getShouldServeCache(shouldServeCache?: boolean | undefined, teamId?: number) {
+  if (typeof shouldServeCache === "boolean") return shouldServeCache;
+  if (!teamId) return undefined;
```

Comment:
Why not `false` here?

Reply:
Undefined means there's no preference. So the regular default behavior will apply.